File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/remote/dto/auth/UserInfoDto.kt
package com.inmotion.in_motion_android.data.remote.dto.auth

import com.inmotion.in_motion_android.data.database.entity.UserInfo

data class UserInfoDto(
    val id: String,
    val email: String,
    val nickname: String,
    val token: String
) {
    fun toUserInfo(): UserInfo {
        return UserInfo(id, email, nickname, token)
    }
}

File: in-motion-server/Auth/IMS.Auth.Models/Dto/Outgoing/UserInfoDto.cs
namespace IMS.Auth.Models.Dto.Outgoing;

public class UserInfoDto
{
    public string Id { get; set; }
    public string Email { get; set; }
    public string Nickname { get; set; }
    public string Token { get; set; }
    public IEnumerable<string> Providers { get; set; }
}